AzureReverseProxy/AzureReverseProxy/AzureReverseProxy/AzureReverseProxy.csproj

```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <None Remove="parameters.json" />
```

Comment:
As these files are not in the PR, the build fails. It's necessary to add them

Reply:
Same as previous comment